src/locales/de.yaml

```diff
@@ -167,7 +172,7 @@ app:
       retract: Retracten
       save: Speichern
       save_as: Speichern als
-      save_restart: Speichern & Neustarten
+      save_restart: Speichern & Neu starten
```

Comment:
Either this or `show_save_config_and_restart` needs to be adjusted so the "Neu starten" / "Neustarten" matches